java-checks/pom.xml

```diff
@@ -128,6 +128,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
+              <runOrder>alphabetical</runOrder>
```

Comment:
The `<runOrder>alphabetical</runOrder>` is already set on `maven-surefire-plugin` of its parent pom and it is inherited. So we don't need to repeat the configuration here.
After removing this line I still have:
```shell
$ cd java-checks
$ mvn -Psanity help:effective-pom | grep runOrder   
              <runOrder>alphabetical</runOrder>
          <runOrder>alphabetical</runOrder>
```